pepdbagent/modules/project.py

```diff
@@ -66,6 +61,7 @@ def get(
         :param name: name of the project (Default: name is taken from the project object)
         :param tag: tag (or version) of the project.
         :param raw: retrieve unprocessed (raw) PEP dict.
+        :param with_id: retrieve project with id [default: False]
```

Comment:
Should we be more specific about what the `id` means? Like, maybe `guid` or `pephub id`? Or something?